src/Media/Audio/AudioPlayer.h

```diff
@@ -145,6 +146,30 @@ class AudioPlayer {
         size_t uDecompressedSize;
     } SoundHeader;
 
+    struct AudioSamplePoolEntry {
+        AudioSamplePoolEntry(PAudioSample samplePtr_, SoundID id_, int pid_):samplePtr(samplePtr_), id(id_), pid(pid_) {}
+
+        PAudioSample samplePtr;
+        SoundID id;
+        int pid;
+    };
+
+    class AudioSamplePool {
```

Comment:
This class is decently encapsulated, let's move it to the global namespace (together with `AudioSamplePoolEntry`).